app/src/components/register/DropZone.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from 'react';
+import * as styles from './DropZone.module.scss';
+import { useDropzone } from 'react-dropzone';
+import { FilesContext } from './RegisterForm';
+import { saveToIPFS } from '../../helper/ipfs';
+
+const DropZone: React.FC<{popup?: true; callback: (hash: string) => void}> = ({ popup, callback }) => {
+  const { files } = React.useContext(FilesContext);
+
+  const onDrop = useCallback((acceptedFiles: Blob[]) => {
+    const reader = new FileReader();
+    reader.addEventListener('loadend', function () {
+      saveToIPFS([reader.result], callback);
+    });
+    acceptedFiles.forEach((blob) => reader.readAsArrayBuffer(blob));
+  }, [callback]);
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+
+  const isFileOver = isDragActive || files.image !== '';
+  const isComplete = files.image !== '';
+
+  return (
+    <>
+      {popup
+        ? <div className={styles.popup}>
+          <span className={styles.message}>
+      Artifact Image<br />
+            <small>Drop your image into the space below</small>
+          </span>
+        </div> : null}
+      <div
+        className={styles.dropzone + (isFileOver ? ' ' + styles.fileOver : '')}
+        {...getRootProps()}
+      >
+        <div className={styles.box}>
+          <div className={styles.progress + (isComplete ? ' ' + styles.complete : '')}></div>
+        </div>
+        <div>
+          <div className={styles.arrow}></div>
+        </div>
+        <img
+          className={styles.imageHolder + (isComplete ? ' ' + styles.move : '')}
+          src={'https://ipfs.io/ipfs/' + files.image}
+          alt='artifact-img'
+        />
+        <input accept="image/*" {...getInputProps()} />
+      </div>
+    </>
+  );
+};
+
+// $(function() {
```

Comment:
Hoping that you intend to either comment this back in and use it or delete it before merging